python/res/enkf/subst_config.py

```diff
@@ -51,23 +51,23 @@ def __init__(self, config_content=None, config_dict=None):
             filename = config_dict.get(ConfigKeys.CONFIG_FILE_KEY)
             if isinstance(filename, str):
                 subst_list.addItem("<CONFIG_FILE>", filename)
-                subst_list.addItem("<CONFIG_FILE_BASE>", filename)
+                subst_list.addItem("<CONFIG_FILE_BASE>", os.path.splitext(filename)[0])
 
             # CONSTANTS #
             constants = config_dict.get(ConfigKeys.DEFINE_KEY)
             if isinstance(constants, dict):
                 for key in constants:
                     subst_list.addItem(key, constants[key])
 
-            # DATA_KW #
+            # DATA_KW
             data_kw = config_dict.get(ConfigKeys.DATA_KW_KEY)
             if isinstance(data_kw, dict):
                 for key, value in data_kw.items():
                     subst_list.addItem(key, value)
 
             # RUNPATH_FILE #
             runpath_file_name = config_dict.get(ConfigKeys.RUNPATH_FILE, ConfigKeys.RUNPATH_LIST_FILE)
-            runpath_file_path = os.path.join(config_directory, runpath_file_name)
+            runpath_file_path = os.path.normpath(os.path.join(config_directory, runpath_file_name))
```

Comment:
Why normapath?

Reply:
The same as before: it compares to abspath. Also when joining two paths might produce `.../...`, which fails in tests.

Reply:
:+1: